projects/openexr/openexr_exrheader_fuzzer.cc

```diff
@@ -0,0 +1,245 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "ImfNamespace.h"
+#include <ImfMultiPartInputFile.h>
+#include <ImfBoxAttribute.h>
+#include <ImfChannelListAttribute.h>
+#include <ImfChromaticitiesAttribute.h>
+#include <ImfCompressionAttribute.h>
+#include <ImfDoubleAttribute.h>
+#include <ImfEnvmapAttribute.h>
+#include <ImfFloatAttribute.h>
+#include <ImfIntAttribute.h>
+#include <ImfKeyCodeAttribute.h>
+#include <ImfLineOrderAttribute.h>
+#include <ImfMatrixAttribute.h>
+#include <ImfPreviewImageAttribute.h>
+#include <ImfRationalAttribute.h>
+#include <ImfStringAttribute.h>
+#include <ImfStringVectorAttribute.h>
+#include <ImfTileDescriptionAttribute.h>
+#include <ImfTimeCodeAttribute.h>
+#include <ImfVecAttribute.h>
+#include <ImfVersion.h>
+#include <ImfHeader.h>
+#include <ImfStdIO.h>
+
+#include <iostream>
+#include <iomanip>
+
+using namespace OPENEXR_IMF_NAMESPACE;
+using namespace std;
+
+void
+printTimeCode (TimeCode tc)
+{
+    setfill ('0');
+#ifndef HAVE_COMPLETE_IOMANIP
+    tc.hours();
+    tc.minutes();
+    tc.seconds();
+    tc.frame();
+#else
+    right << tc.hours();
+    right << tc.minutes();
+    right << tc.seconds();
+    right << tc.frame();
+#endif
+    setfill (' ');
+    tc.dropFrame();
+    tc.colorFrame();
+    tc.fieldPhase();
+    tc.bgf0();
+    tc.bgf1();
+    tc.bgf2();
+    tc.userData();
+}
+
+void
+printChannelList (const ChannelList &cl)
+{
+    for (ChannelList::ConstIterator i = cl.begin(); i != cl.end(); ++i)
+    {
+        i.name();
+        i.channel();
+    }
+}
+
+
+void
+printInfo (IStream &is)
```

Comment:
We're not actually printing anything now, so let's rename this to `dumpInfo` or something like that.

Reply:
Same for `printChannelList` above.